bundles/automation/org.eclipse.smarthome.automation.java.api.demo/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,20 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Automation Java API Demo - Welcome Home Application
+Bundle-SymbolicName: org.eclipse.smarthome.automation.java.api.demo
+Bundle-Activator: org.eclipse.smarthome.automation.java.api.demo.Activator
+Bundle-Version: 0.8.0.qualifier
+Bundle-Vendor: Eclipse.org/SmartHome
+Import-Package: 
+ org.apache.commons.lang;version="2.6.0",
```

Comment:
It is used because of the StringUtils.join(getUsages(), "\n"));
( org.apache.commons.lang.StringUtils )
If you think there is license issue, then this code can be replaced